releng/tools/_extract_rust_executable.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+
+import sys
+import json
+
+
+executable = None
+
+for line in sys.stdin:
+    artifact = json.loads(line)
+    if artifact['reason'] != 'compiler-artifact':
+        continue
+    if artifact['executable'] is None:
+        continue
+
+    if executable is not None:
+        raise RuntimeError("Multiple executables")
+    executable = artifact['executable']
+
+if executable is None:
+    raise RuntimeError("No executables")
+
+print(executable)
```

Comment:
Hloupost (která je možná už zanesena i na jiných místech porušuje), ale preferoval bych zabalené tělo do `if __name__ == '__main__` a samotnou funkcionalitu hodit do funkce. Čistě pro to, kdyby to někoho napadlo importovat.